content/2020-03-05_koucovani-na-pyladies_kurzech.md

```diff
@@ -0,0 +1,41 @@
+Title: Koučování na Pyladies kurzech
+Date: 2020-03-05 11:00:00
+Author: Iveta Česalová
+Gravatar: cesalova.iveta@seznam.cz
+About: Iveta je bývalá účetní, má dvě děti a po absolvování několika Pyladies kurzů se jí podařilo získat vysvěnou práci v IT.
+
+*Text vyšel původně na [autorčině blogu](https://ivet1987.wz.cz/2020/03/koucovani-na-pyladies-kurzech/).*
+
+Nevím, zda jsem ta správná osoba, která o tom může vyprávět, ale taky přispěji svou troškou do mlýna. 
+Jako vedlejší koučka na začátečnických kurzech jsem byla zatím pouze jeden semestr. 
+Víc čas nedovolil, když jsem měla další kurzy. Ale byla to velmi příjemná zkušenost.
+
+**Proč začít koučovat?** Co k tomu lidi vede? Někoho baví učit, někdo se chce odvděčit pyladies, 
+například absolventky začátečnických kurzů a předat nabyté znalosti dál. Jiní si tam chodí upevňovat základy, 
+protože nemusíte každou hodinu jen koučovat, ale být tam jako pozorovatel a opakovat si látku, ve které si nejste jistí. 
+A největší motivace – šířit Python! 😀 Ale hlavně ta milá společnost na kurzech a příjemné prostředí, 
+celá python komunita je neskutečná, tu nemůžete nemilovat.
+
+Moje **motivace** byla kombinací výše zmíněných. Co bylo nejdůležitější, tak ta atmosféra na kurzech, 
+chodila jsem si tam odpočinout. Nevadí různé všetečné otázky, protože jak víte, žádná otázka není hloupá! 🙂 
+Není tam žádný stres, spěch, tempo se přizpůsobí vždy individuálně. Každý je ochoten pomoci se sebemenším problémem. 
+Přeci jen nejblíže k účastníkům kurzů mají právě ostatní absolventky a můžou jim případně vysvětlit látku více lidsky. 
+Tím nechci říct, že by hlavní kouč přednášel nesrozumitelně, ale je to přeci jen programátor profesionál a my, 
+obyčejní smrtelníci to asi vysvětlíme úplně laicky neaťácky 🙂 Někdy jsem si všimla, že holky tápou, nezeptají se, 
+i když neví. Není nic jednoduššího, než se zeptat sama, i když to vlastně víte, ale je super, že to pak pochopí i ostatní.
+
+Na začátku to byly hrozné pocity, že si mě někdo zavolá a bude mít problém, se kterým nedokáži pomoci, 
+a budu za trubku. No jistě, občas taková situace nastala, ale jednoduše se zavolal druhý, více zkušenější kouč, 
+a byli jsme chytřejší všichni 🙂 Ale nakonec to nebylo tak složité. Jistě, objevil se červený lísteček, znamení pro kouče, 
```

Comment:
Podruhé "ale" za sebou mi moc nezní, ale to už bych moc rejpal do stylistyky. Klidně můžeme nechat takto.